lib/soapy_cake/admin.rb

```diff
@@ -109,11 +109,16 @@ def update_campaign(opts = {})
     def add_blacklist(opts)
       require_params(opts, %i(blacklist_date))
 
+      blacklist_date = opts[:blacklist_date]
```

Comment:
What if this is a `DateTime`? Also, the api doc says it has to be in the format `MM-DD-YYYY`, that's probably not what `Date#to_s` does, right?

See: https://support.getcake.com/support/solutions/articles/5000593922-addedit-blacklist-api-version-1